firmware/src/system_config/number8/system_init.c

```diff
@@ -103,111 +103,6 @@ SUBSTITUTE GOODS, TECHNOLOGY, SERVICES, OR ANY CLAIMS BY THIRD PARTIES
 // *****************************************************************************
 // *****************************************************************************/
 
-//<editor-fold defaultstate="collapsed" desc="USB Stack Configuration">

```

Comment:
It's boilerplate that gets added. Rather than duplicate the descriptor config across each of the system_init.c files it lives in usb_descriptors.c